src/data/interfaces/AccidentData.ts

```diff
@@ -3,7 +3,9 @@
 
 export type AccidentData = {
   ID: number; // id
-  Starttijd: Date; // start date, access value with: accidents.current[i].Startdatum.toString()
+  Starttijd: Date; // start time, access value with: accidents.current[i].Startdatum.toString()
+  Eindtijd: Date; // start date
```

Comment:
I think the comments for Eindtijd and Startdatum need to be switched?

Also may I ask what the difference is between Starttijd and Startdatum

Reply:
So I tried to make the current AccidentData model compatible with both Brabant and RWS.
- For Brabant, the full dates (date+time) are stored in fields Starttijd and Einddatum, these are the only relevant data-related fields in that excel. 
- For RWS, the time (hour, minute, second) is stored in Starttijd and Eindtijd, while the dates (day, month, year) are stored in Startdatum and Einddatum. The comments are a bit messed up, I fixed them now:)